eng/Versions.props

```diff
@@ -48,7 +48,7 @@
     Versions managed by Arcade (see Versions.Details.xml)
   -->
   <PropertyGroup>
-    <SystemCommandLineVersion>2.0.0-beta5.25210.1</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta5.25224.104</SystemCommandLineVersion>
```

Comment:
Ominous

Reply:
Yes, we might revert this change. I believe we should flow command-line-api not via the VMR but externally via the existing subscription as it keeps its official build.